tests/actor/test_actor.py

```diff
@@ -18,20 +18,26 @@
 from dapr.serializers import DefaultJSONSerializer
 
 from tests.actor.fake_actor_classes import (
+    FakeDaprActorClient,
     FakeSimpleActor,
     FakeSimpleReminderActor,
     FakeSimpleTimerActor,
     FakeMultiInterfacesActor,
 )
 
+from tests.actor.utils import (
+    _async_mock,
+    _run
+)
+
 
-class ActorTests(unittest.IsolatedAsyncioTestCase):
-    async def asyncSetUp(self):
+class ActorTests(unittest.TestCase):
+    def setUp(self):
         ActorRuntime._actor_managers = {}
         ActorRuntime.set_actor_config(ActorRuntimeConfig())
         self._serializer = DefaultJSONSerializer()
-        await ActorRuntime.register_actor(FakeSimpleActor)
-        await ActorRuntime.register_actor(FakeMultiInterfacesActor)
+        _run(ActorRuntime.register_actor(FakeSimpleActor))
```

Comment:
use asyncio.run()

Reply:
As discussed offline asyncio.run() can not be used more than once in the same thread and should only be used as a main entry point.